src/rpc/blockchain.cpp

```diff
@@ -2108,3 +2108,66 @@ UniValue rpc_reorganize(const UniValue& params, bool fHelp)
     results.pushKV("RollbackChain",fResult);
     return results;
 }
+
+UniValue getburnreport(const UniValue& params, bool fHelp)
+{
+    if (fHelp || params.size() != 0)
+        throw runtime_error(
+                "getburnreport\n"
+                "Scan for and aggregate amounts for provably-destroyed outputs.\n");
```

Comment:
Might be good to clarify that this is network wide and not just the aggregate amount burned for transactions from your wallet

Reply:
That's true. It is listed in the "network" RPC category, though. I can add that bit.

Reply:
Done.